### PR TITLE
Add Docs for Risc0 adapter

### PR DESCRIPTION
**Pull request by citizen-stig**
# Description
Adds docs to Risc0 adapter

## Linked Issues
- Related to #438

## Testing
No testing

## Docs
Added docs

### CODE REVIEW

File: adapters/risc0/src/host.rs [hunk 1]
```diff
@@ -9,12 +9,14 @@ use sov_rollup_interface::zk::{Zkvm, ZkvmHost};
...
 use crate::Risc0MethodId;
 
+/// Allows to run prover code and send data to ZkVM
 pub struct Risc0Host<'a> {
     env: RefCell<ExecutorEnvBuilder<'a>>,
     elf: &'a [u8],
 }
 
 impl<'a> Risc0Host<'a> {
+    /// Create a new host fo a given reference to an ELF file
```
Comment (preston-evans98) [inline adapters/risc0/src/host.rs]: nit: It's not obvious what it means that a host is "for a given reference to an ELF file". Maybe "Create a host to prove the provided ELF file?"

File: adapters/risc0/src/host.rs [hunk 2]
```diff
@@ -57,6 +59,7 @@ impl<'prover> Zkvm for Risc0Host<'prover> {
     }
 }
 
+/// Making verification of Risc0 proofs
```
Comment (preston-evans98) [inline adapters/risc0/src/host.rs:86]: nit: This english isn't quite right. Maybe "Verifies Risc0 proofs"?

File: adapters/risc0/src/host.rs [hunk 3]
```diff
@@ -9,12 +9,14 @@ use sov_rollup_interface::zk::{Zkvm, ZkvmHost};
 
 use crate::Risc0MethodId;
 
+/// Allows to run prover code and send data to ZkVM
```
Comment (preston-evans98) [inline adapters/risc0/src/host.rs]: nit: This comment is a little terse. How about...

"The 'host' of a Risc0 zkVM. Generates a SNARK by running the target code on a virtual RISC-V CPU. The host is responsible for sending the "witness" data to the zkVM guest."

**Review by neysofu — COMMENTED** [review]
Just a nitpick: doc comments should always have a period at the end of sentences. It really adds to the perceived quality of the text when looking at docs on [docs.rs](https://docs.rs).

**Comment by citizen-stig** [comment]
Closed in favour of #934

**Pull request closed without merging**